blocks/library/code-mirror/index.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * External dependencies
+ */
+// when this CSS was imported dynamically
+// it was causing major visual glitches to
+// the editor which completely broke it
+require( 'codemirror/lib/codemirror.css' );
```

Comment:
All styles are loaded using the following syntax:

```js
import './style.scss';
```

Webpack uses the following pattern `test: /style\.s?css$/,` to match styles. So it should work here, too. Unless this is the reason why it breaks :)

Reply:
actually I think it had to do with race conditions on the mounted components. I was comparing to async loading as the dynamic bits though vs. `import` statement.

mainly I meant to convey that this stylesheet seemed to need to load before the editor and I wasn't sure how best to do that